canopen_chain_node/include/canopen_chain_node/chain_ros.h

```diff
@@ -327,6 +327,15 @@ template<typename InterfaceType> class RosChain : public canopen::LayerStack {
                 return false;
             }
 
+            ObjectDict::Overlay overlay;
+            if(module.hasMember("eds_overlay")){
```

Comment:
@ipa-mdl Here the parameter is "eds_overlay", but on https://github.com/ipa-mdl/ros_canopen/blob/dcf_overlay/canopen_test_utils/config/canopen_rig1.yaml it is called dcf_overlay. Which one is correct?